components/ai_chat/resources/page/components/model_selector/index.tsx

```diff
@@ -0,0 +1,214 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import ButtonMenu from '@brave/leo/react/buttonMenu'
+import Button from '@brave/leo/react/button'
+import Icon from '@brave/leo/react/icon'
+import Label from '@brave/leo/react/label'
+import classnames from '$web-common/classnames'
+import * as Mojom from '../../../common/mojom'
+import { getLocale } from '$web-common/locale'
+import { useAIChat } from '../../state/ai_chat_context'
+import { useConversation } from '../../state/conversation_context'
+import { getModelIcon } from '../../../common/constants'
+import styles from './style.module.scss'
+
+interface Props {}
+
+export function ModelSelector(props: Props) {
```

Comment:
nit: lets just remove the empty props interface
```suggestion
export function ModelSelector() {
```

Reply:
oof.. not sure how I let that one slip 🤦  Removed!